addons/finger/ACE_Settings.hpp

```diff
@@ -0,0 +1,23 @@
+#include "code\addon.hpp"
+
+#define LOC(x) $__LOCALIZE_PREFIX__##x
+
+
+class ACE_Settings {
+	class GVAR(FP_indicator_for_self) {

```

Comment:
The FP prefix should be removed everywhere. Settings should be in camel case like `GVAR(indicatorForSelf)`